core/src/apps/benfen/tx_parser.py

```
@@ -0,0 +1,432 @@
+            inputs.append(
+                ParsedInput(
+                    kind="Input",
+                    value={
+                        "type": "ImmOrOwnedObject",
+                        "objectId": f"0x{binascii.hexlify(obj_ref.address.data).decode()}",
+                        "sequenceNumber": obj_ref.sequence_number,
+                        "digest": binascii.hexlify(obj_ref.digest).decode(),
+                    },
+                    index=i,
+                    type="object",
+                )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)